src/test/java/homework_3/Constants.java

```diff
@@ -0,0 +1,53 @@
+package homework_3;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by dinar on 23.11.2019.
+ */
+public class Constants {
+    //header menu
+    public final static String HOME = "Home";
+    public final static String CONTACT_FORM = "Contact form";
+    public final static String SERVICE = "Service";
+    public final static String METALS_AND_COLORS = "Metals & Colors";
+    //images from page
+    public final static String PRACTISE_FIRST = "To include good practices\n" +
+            "and ideas from successful\n" +
+            "EPAM project";
+
+    public final static String CUSTOM_SECOND = "To be flexible and\n" +
+            "customizable";
+
+    public final static String MULTI_THIRD = "To be multiplatform";
+
+    public final static String BASE_FOURTH = "Already have good base\n" +
+            "(about 20 internal and\n" +
+            "some external projects),\n" +
+            "wish to get more…";
+    public final static String TEXT_UNDER_MAIN_TITLE = "LOREM IPSUM DOLOR SIT AMET, CONSECTETUR " +
+            "ADIPISICING ELIT, SED DO EIUSMOD TEMPOR INCIDIDUNT UT LABORE ET DOLORE MAGNA ALIQUA." +
+            " UT ENIM AD MINIM VENIAM, QUIS NOSTRUD EXERCITATION ULLAMCO LABORIS NISI UT ALIQUIP" +
+            " EX EA COMMODO CONSEQUAT DUIS AUTE IRURE DOLOR IN REPREHENDERIT IN VOLUPTATE" +
+            " VELIT ESSE CILLUM DOLORE EU FUGIAT NULLA PARIATUR.";
+
+    public final static String DIFFER_ELEMENETS = "Different Elements";
+    public final static List<String> listDropMenuService =
```

Comment:
Please follow Java Code Convention for the `constants` names

Reply:
fix